client/src/components/UpcomingTripsCard/UpcomingTripsCard.js

```diff
@@ -0,0 +1,32 @@
+
+import React from 'react';
+import "./UpcomingTripsCard.css";
+import { Link } from "react-router-dom";
+import Card from "../Card/Card";
+import TripCard from "../TripCard/TripCard";
+import img from "../../assets/images/card_small2.png";
+
+const UpcomingTripsCard = (props) => {
+
+  let details = props.details;
+
+  return(
+    <Card color="pink" className="UpcomingTripsCard">
+      <header>{props.header}</header>
+      <div className="UpcomingTripsCard__info">
+        <TripCard src={img} location="Puerto Rico" date="April 2019"/>
```

Comment:
They may want to pass in this information from props (the image, location, and date)

Reply:
By "they", I mean "we"